perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -55,18 +54,20 @@
 cassandra_stress:
   description: Benchmark Cassandra using cassandra-stress
   vm_groups:
-    default:
+    cassandra:
+      vm_spec: *default_single_core
+      disk_spec: *default_500_gb
```

Comment:
This was discussed in #510; the decision there was to leave the current default in place and be explicit about the characteristics of the disks involved. @noahl - could you open an issue for your proposed changes to this?